File: SkuciSeCode/SkuciSeCode/Entities/Appointment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkuciSeCode.Entities
{
    public class Appointment
    {
        public int id { get; set; }
        public int user_id { get; set; }
        public int ad_id { get; set; }
        public String date { get; set; }
        public int approved { get; set; }

        public Appointment()
        {

        }
        public Appointment(int id, int user_id, int ad_id, string date, int approved)
        {
            this.id = id;
            this.user_id = user_id;
            this.ad_id = ad_id;
            this.date = date;
            this.approved = approved;
        }

        public Appointment(int user_id, int ad_id, string date, int approved)
        {
            this.user_id = user_id;
            this.ad_id = ad_id;
            this.date = date;
            this.approved = approved;
        }
    }
}

File: SkuciSe/app/src/main/java/com/example/myapplication/data/model/Appointment.java
package com.example.myapplication.data.model;

public class Appointment
{
    private final int id;
    private final int user_id;
    private final int ad_id;
    private final int approved;
    private final String date;

    public Appointment(int id, int user_id, int ad_id, int approved, String date) {
        this.id = id;
        this.user_id = user_id;
        this.ad_id = ad_id;
        this.approved = approved;
        this.date = date;
    }

    public int getId() {
        return id;
    }

    public int getUser_id() {
        return user_id;
    }

    public int getAd_id() {
        return ad_id;
    }

    public int getApproved() {
        return approved;
    }

    public String getDate() {
        return date;
    }
}
